src/attack.c

```diff
@@ -339,6 +341,89 @@ check_new_vhosts (void)
   unset_check_new_vhosts_flag ();
 }
 
+/**
+ * @brief Publish the necessary data to start a Table driven LSC scan.
+ *
+ * If the gather-package-list.nasl plugin was launched, and it generated
+ * a valid package list for a supported OS, the table driven LSC scan
+ * which is subscribed to the topic will perform a scan an publish the
+ * the results to be handle by the sensor/client.
+ */
+static void
+run_table_driven_lsc (const char *scan_id, kb_t kb, const char *ip_str,
```

Comment:
I would move all the json handling to plugutils.c. Maybe the function could be calles something like `scanner_get_table_driven_lsc_start_str(...);` and be placed at the top where the other `scanner_..` function is. /misc already has json support, so no changes to the cmake needed.

The  `run_table_driven_lsc` function can stay in attack.c I think.